vNext/src/ClientGenerator/BootstrapCodegen.proj

```diff
@@ -1,39 +1,79 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
```

Comment:
That did not change, for a reason. ToolsVersion is set to 15.0 to make sure latest tools are used. For CSProj, VBProj, etc you're right, that's what I changed when moved to new tooling.

I've a some thoughts about how to change the bootstrapping but removing commons import and having an Sdk reference on the project node  is now working in itself.